deploy/common/deploy.sh

```diff
@@ -108,6 +108,34 @@ function wait_for_central {
     echo
 }
 
+# Checks if central already exists in this cluster.
+# If yes, the user is asked if they want to continue. If they answer no, then the script is terminated.
+function check_central_exists() {
+    central_pod="$("${ORCH_CMD}" -n stackrox get pods -l app=central -ojsonpath='{.items[0].metadata.name}')"
+    if [[ -n "$central_pod" ]]; then
+        yes_no_prompt "Detected there is already a central running on this cluster. Are you sure you want to proceed with this deploy?" || { echo >&2 "Exiting as requested"; exit 1; }
+    fi
+}
```

Comment:
I think it's better to put these new functions in `k8sbased.sh` instead of `deploy.sh` because similar functions are already in `k8sbased.sh`: `verify_orc` and `local_dev`. And besides there's nothing in `deploy.sh` that calls them.